compiler/rustc_session/src/options.rs

```diff
@@ -1986,10 +2221,10 @@ options! {
         "enable queries of the dependency graph for regression testing (default: no)"),
     randomize_layout: bool = (false, parse_bool, [TRACKED],
         "randomize the layout of types (default: no)"),
-    reg_struct_return: bool = (false, parse_bool, [TRACKED],
+    reg_struct_return: bool = (false, parse_bool, [TRACKED TMOD],
```

Comment:
I think it would better if `TMOD` was `TARGET_MODIFIER` in these macros.

Reply:
Fixed.